props.json

```diff
@@ -7,6 +7,30 @@
         "title": "Spring",
         "thumbnail": "https://igalia.github.io/wolvic-3D-environments/spring/spring.png",
         "payload": "https://igalia.github.io/wolvic-3D-environments/spring/spring.zip"
+      },
```

Comment:
Hmm no, we better create a new entry for 1.5.2 and add them there

Reply:
I think we should add the new envs in the "official" repo 

https://github.com/Igalia/wolvic-3D-environments

Reply:
Done.

Reply:
Please ensure that we need to repeat the 1.5.1 entry here